README.rst

```diff
@@ -107,6 +107,37 @@ Installing this package will install ``nvidia-dali`` and its dependencies, if th
 
 ----
 
+Compiling DALI from source (Docker) - recommended
+-------------------------------------------------
+
+Prerequisites
+^^^^^^^^^^^^^
+
+.. |docker link| replace:: **Docker**
+.. _docker link: https://docs.docker.com/install/
+
+.. table::
+   :align: center
+
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | Linux x64                              |                                                                                             |
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | |docker link|_                         | Follow installation guide and manual at the link.                                           |
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+
+Building Docker image
+^^^^^^^^^^^^^^^^^^^^^
+
+Change directory (``cd``) into Docker directory and run ``./build.sh``. If needed, set the following environment variables:
+
+* PYVER - Python version. Default is ``2.7``.
+* CUDA_VERSION - CUDA toolkit version. Default is ``10``.
+* NVIDIA_BUILD_ID - Custom ID of the build. Default is ``1234``.
+* CREATE_RUNNER - Create Docker image with cuDNN, CUDA and DALI installed inside. It will create the ``Docker_run_cuda`` image, which needs to be run using ``nvidia-docker`` and DALI wheel in the ``wheelhouse`` directory under$
+* CREATE_WHL - Create a wheel also. Default is ``YES``.
```

Comment:
Could we expand this slightly to make it less ambiguous? `Create a wheel usable outside of the container` or something like that?
Also maybe we could say something at the beginning that with this method one can obtain the whl similar to the one we post.

Reply:
Done (I hope)